doc/rst/locks.rst

```diff
@@ -0,0 +1,45 @@
+################################
+lib_locks: Lock handling Library
```

Comment:
Capitalisation issue

Reply:
Or should Library be capitalised?

Reply:
I think handling needs capitalisation? looking at other examples...